.appveyor.yml

```diff
@@ -315,6 +315,8 @@ install:
         )
       }
 
+  - rustup default 1.30.0
```

Comment:
If you do just this, then on every CI run the following will happen once rust 1.31.0 comes out:

* `rustup update` will try to upgrade. After a while this fails because the docs have been deleted.
* Rust gets removed.
* rustup init installs rust v1.31.1
* rustup default additionally installs rust v1.30.1

I think that't not ideal. 

Instead, I would suggest:
* Remove the section that upgrades rust.
* Change the install command to `rustup-init --default-toolchain 1.29.0`
...and call it a day.

Reply:
Updated.. Is this what you were suggesting?

Reply:
Good start.
Also remove this: https://github.com/denoland/deno/blob/a36ed6805ec6825bac7b6c6a66e1d6e1b92c289c/.appveyor.yml#L286-L299